### ISSUE TITLE
Parsing error: Expected value to be truthy

### ISSUE THREAD

**Issue by vb** [mdx-js/eslint-mdx#462]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

eslint-plugin-mdx 2.1.0

### Link to runnable example

https://github.com/vb/eslint-plugin-mdx-tabs-spaces

### Steps to reproduce

- Clone `https://github.com/vb/eslint-plugin-mdx-tabs-spaces`
- `npm install`
- `npm run lint`

### Expected behavior

Both mdx files should lint without errors

### Actual behavior

eslint throws error on file `spaces.mdx`

```bash
spaces.mdx
  0:0  error  Parsing error: Expected value to be truthy

✖ 1 problem (1 error, 0 warnings)
```

### Runtime

Node v18

### Package manager

npm v9

### OS

macOS

**Comment by vb** [comment]
Some additional info:
The new version 2.1.0 (and older versions afaik) throws the error `Parsing error: Expected value to be truthy` when using spaces instead of tabs when passing an object as prop in JSX.

```jsx
<button 
  style={{
    background: 'red',
    border: '1px solid red',
  }}
>
  Hello
</button>
```

Minimal reproducible example is available here https://github.com/vb/eslint-plugin-mdx-tabs-spaces

**Comment by JounQin** [comment]
duplicate of #435

**Comment by vb** [comment]
You're right and sorry for the duplicate! I did not see the other ticket
